src/opm/input/eclipse/Schedule/KeywordHandlers.cpp

```diff
@@ -1474,6 +1490,21 @@ File {} line {}.)", wname, location.keyword, location.filename, location.lineno)
 
             this->addWellToGroup(groupName, wellName, handlerContext.currentStep);
         }
+
+        if (! fieldWells.empty()) {
+            const auto& location = keyword.location();
+
+            auto wellNames = std::string{};
```

Comment:
Out of curiosity: what is the reason to prefer `auto` and not
```
std::string wellNames;
```
or
```
std::string wellNames{};
```

Reply:
> Out of curiosity: what is the reason to prefer `auto` and not
>
> [traditional style]

Mostly for uniformity.  I like that the `auto` style brings the object closer to the start of the line.  That said, I read the [documentation](https://fmt.dev/latest/api.html) for libfmt more closely and noticed the `fmt::join()` function which performs the task a lot better than manual code so I've switched to using that.  This function is also slated to become part of C++23 for that matter.